tools/ci_generate.nim

```diff
@@ -102,6 +102,51 @@ nimBuildCsourcesIfNeeded "$@"
 {genBuildExtras("echo_run", "./koch", "bin/nim")}
 """
 
+proc genDroneScript(): string =
+  const Image = "ghcr.io/alaviss/nim-ci:sha-87f5a84"
+  type
+    Arch = enum
+      arm64
+      arm
+
```

Comment:
```suggestion
  result = fmt"# {doNotEdit}"
```
(replace `#` with whatever is recognized as comment marker)

see how other pipelines use this so that ppl know where the generated code comes from and are not tempted to edit the generated code directly;

Reply:
If you notice the latest commit, I had to remove it. It doesn't seem to play well with drone yaml?

Reply:
* well the last commit also adds `nim c koch` so it's unclear it's due to that comment?

* where can i see CI logs? if i click on links inside https://cloud.drone.io/alaviss/Nim/ it shows 404's

* how about using yaml syntax instead of json, since our other pipelines use yaml; eg see https://github.com/hbokh/docker-drone-builder/blob/master/.drone.yml (which btw also contains `#` for comments)

Reply:
> where can i see CI logs? if i click on links inside https://cloud.drone.io/alaviss/Nim/ it shows 404's

Hmm, not sure, it used to work without logging in.

> how about using yaml syntax instead of json, since our other pipelines use yaml and it's arguably more readable; eg see https://github.com/hbokh/docker-drone-builder/blob/master/.drone.yml (which btw also contains # for comments), or pretty much most drone.yml files one can find on github

It's easier to build with json, that's all. And the code is not even that complicated...

> should it be drone.yml or .drone.yml ? (i honestly don't know)

we can name it whatever, you get to set it on repository setup